src/data/priv_appendable_data.rs

```diff
@@ -384,296 +323,256 @@ mod test {
+    //         Ok(mut priv_appendable_data) => {
+    //             // After one signature, one more is required to reach majority.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys1.1)), 1);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys)
+    //                 .is_err());
+    //             // Two out of three is enough.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys2.1)), 0);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys).is_ok());
+    //             // Three out of three is also fine.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys3.1)), 0);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys).is_ok());
+    //         }
+    //         Err(error) => panic!("Error: {:?}", error),
+    //     }
+    // }
+
+    // // TODO: this test is disabled as the feature of multi-sig is disabled
+    // #[ignore]
+    // #[test]
+    // fn four_owners() {
+    //     let keys1 = sign::gen_keypair();
+    //     let keys2 = sign::gen_keypair();
+    //     let keys3 = sign::gen_keypair();
+    //     let keys4 = sign::gen_keypair();
+    //     let encrypt_keys = box_::gen_keypair();
+
+    //     let owner_keys = vec![keys1.0, keys2.0, keys3.0, keys4.0];
+
+    //     match PrivAppendableData::new(rand::random(),
+    //                                   0,
+    //                                   owner_keys.clone(),
+    //                                   vec![],
+    //                                   BTreeSet::new(),
+    //                                   Filter::white_list(None),
+    //                                   encrypt_keys.0,
+    //                                   Some(&keys1.1)) {
+    //         Ok(mut priv_appendable_data) => {
+    //             // Two signatures are not enough because they don't have a strict majority.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys2.1)), 1);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys).is_ok());
+    //             // Three out of four is enough.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys3.1)), 0);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys).is_ok());
+    //             // Four out of four is also fine.
+    //             assert_eq!(unwrap!(priv_appendable_data.add_signature(&keys4.1)), 0);
+    //             assert!(priv_appendable_data.verify_previous_owner_signatures(&owner_keys).is_ok());
+    //         }
+    //         Err(error) => panic!("Error: {:?}", error),
+    //     }
+    // }
+
+    // // TODO: this test is disabled as the feature of multi-sig is disabled
+    // #[ignore]
+    // #[test]
+    // fn transfer_owner_attack() {
```

Comment:
Do we have a test for a valid ownership transfer with a single owner? We probably should.